### DISCUSSION TITLE
Do open issues ever get looked at?

### DISCUSSION THREAD

**Discussion by ToshBimbra** [codefori/vscode-ibmi#1759] [Q&A]
[CRRDG1121E](https://github.com/codefori/vscode-ibmi/issues/1699)
This has been idle and getting no response.

**Comment by worksofliam** [comment]
I'm on vacation (for the last 3 weeks) and none of the core team (including me) tend to solve debug issues outside of our docs easily since it is maintained by an IBMer.

@mkwan01 should be able to help when he is available.

**Comment by sebjulliand** [comment]
You may want to tone it down, too, @ToshBimbra .
A quick look at the repositories could tell you that yes, all issues are being looked at and fixed whenever it's possible. Keep in mind this is a community project on which people work on their spare time.
The source code is available to everybody, if you want to figure out if something is wrong by yourself. As far as I can tell, that issue is related to the certificates that are being generated by the walkthrough. Nothing prevents you from manually recreate these steps and see if it's an IBM problem or a Code for i issue. That would be way more helpful than opening that kind of discussion.
2024 is still young, it's not too late to add `being nice and helpful` to your new year's resolutions 😉
